pkg/consts/consts.go

```diff
@@ -113,8 +113,8 @@ const (
 	// StrRawVersion is the raw version string
 	StrRawVersion string = "raw"
 
-	// VirtualMachineScaleSetsDeallocating indicates VMSS instances are in Deallocating state.
-	VirtualMachineScaleSetsDeallocating = "Deallocating"
```

Comment:
This is wrong because the Deallocating is powerstate instead of provisioning state. The correct state is Deleting.

Reply:
Good catch!